src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -130,6 +140,10 @@ public OAuthProxyFactory(
         this.lookupAuthorization = lookupAuthorization;
         this.executor = executor;
         this.httpTypeId = supplyTypeId.applyAsInt("http");
+        for(int i = 0; i < grantsBySubjectByAffinityPerRealm.length; i++)
+        {
+            grantsBySubjectByAffinityPerRealm[i] = new Long2ObjectHashMap<>();
+        }
```

Comment:
Suggest using `Arrays.setAll` instead of explicit `for` loop.

```
Arrays.setAll(grantsBySubjectByAffinityPerRealm, i -> new Long2ObjectHashMap());
```